### ISSUE TITLE
v4.15 google play console: NullPointerException (ZipFile.getInputStream)

### ISSUE THREAD

**Issue by emulienfou** [nova-video-player/aos-AVP#231]
Hello there, it seems there is a Fatal error when Nova player tries to index some content from SMB server.

Here is the logcat output I was able to retrieve:
```log
11-15 21:10:42.692 31725 32009 E AndroidRuntime: FATAL EXCEPTION: Thread-15
11-15 21:10:42.692 31725 32009 E AndroidRuntime: Process: org.courville.nova, PID: 31725
11-15 21:10:42.692 31725 32009 E AndroidRuntime: java.lang.NullPointerException: entry
11-15 21:10:42.692 31725 32009 E AndroidRuntime: 	at java.util.zip.ZipFile.getInputStream(ZipFile.java:372)
11-15 21:10:42.692 31725 32009 E AndroidRuntime: 	at com.archos.mediascraper.xml.ShowScraper2.getDetailsInternal(SourceFile:173)
11-15 21:10:42.692 31725 32009 E AndroidRuntime: 	at com.archos.mediascraper.xml.BaseScraper2.getDetails(SourceFile:145)
11-15 21:10:42.692 31725 32009 E AndroidRuntime: 	at com.archos.mediascraper.xml.BaseScraper2.search(SourceFile:107)
11-15 21:10:42.692 31725 32009 E AndroidRuntime: 	at com.archos.mediascraper.Scraper.getAutoDetails(SourceFile:102)
11-15 21:10:42.692 31725 32009 E AndroidRuntime: 	at com.archos.mediascraper.AutoScrapeService$3.run(SourceFile:351)
11-15 21:10:42.713 31725 32009 I Process : Sending signal. PID: 31725 SIG: 9
```

**Comment by courville** [comment]
Thanks for reporting.
Indeed this seems to be due to change on thetvdb servers.

**Comment by courville** [comment]
Could you please try this apk that perhaps will solve the crash you have (upgrade to new thetvdb-java lib and change of the showscraper): https://drive.google.com/open?id=1NYxB2nKtU50y4HEXAw1p7g-lxUXPLxgV
Please note that  this will not solve the tv series scraping issue that comes from the thetvdb.com site recent changes.

**Comment by emulienfou** [comment]
Hi @courville, thanks for this APK. This one fix the crash issue I previously had.

**Comment by emulienfou** [comment]
Issue fixed from v4.18, thanks @courville for this release and fix.
